packages/backend/src/api/endpoints/AccountlessApplicationsAPI.ts

```diff
@@ -5,17 +5,26 @@ import { AbstractAPI } from './AbstractApi';
 const basePath = '/accountless_applications';
 
 export class AccountlessApplicationAPI extends AbstractAPI {
-  public async createAccountlessApplication() {
+  public async createAccountlessApplication(headers: Headers | undefined) {
+    const headerParams = headers instanceof Headers ? Object.fromEntries(headers.entries()) : undefined;
     return this.request<AccountlessApplication>({
       method: 'POST',
       path: basePath,
+      headerParams,
     });
   }
 
-  public async completeAccountlessApplicationOnboarding() {
+  public async completeAccountlessApplicationOnboarding(headers: Headers | undefined) {
```

Comment:
Let's change to optional object instead

```suggestion
  public async completeAccountlessApplicationOnboarding(params?: { requestHeaders?: Headers }) {
```